src/engine/uct/node/mod.rs

```diff
@@ -146,21 +146,38 @@ impl Node {
                 .iter()
                 .map(|&m| Node::new(m, self.config))
                 .collect();
+            if self.children.len() <= (game.size() * game.size() / 10) as usize {
+                let player = game.last_move().color().opposite();
+                if game.winner() == player || !self.config.uct.playout_aftermath { //don't pass if we're losing on the board
```

Comment:
I **still* don't see why we need the ability to turn it off (or rather why we need it here, but not in `expand()` and `remove_illegal_children()`). After all our scoring algorithm is identical to the one used by CGOS. 

Sorry for being so difficult. I just want to understand why we need this.